tests/rados/test_health_warns.py

```diff
@@ -100,6 +100,102 @@ def common_workflow(param_dict: dict):
 
                 assert mon_obj.remove_config(section="osd", name=key)
 
+            # begin verification of precedence for the input parameters across config levels and MON and MGR daemons
+            log.debug(
+                "Begin verification of precedence for the input"
+                " parameters across config levels and MON, MGR daemons"
+            )
+            value_map = {}
+            for key in param_dict.keys():
+                for level in ["global", "osd", "host", "daemon", "mon", "mgr"]:
```

Comment:
The test cases CEPH-83620333 and  CEPH-83620336 need to update as per the automation steps.
Verification of default values and   precedence details are missing in the test cases.